atlas/tasks.py

```diff
@@ -298,6 +352,14 @@ def code_remove(item, other_static_assets=True):
             utilities.code_type_directory_name(item['meta']['code_type']),
             item['meta']['name'])
         os.unlink(code_folder_current)
+        # Remove symlink for versions of package into the default profiles
```

Comment:
The pattern of querying for profiles and linking the item if there are profiles is repeated a few times in this file, it might be worth moving into `code_operations.update_symlink_profile` to reduce duplication